README.md

```diff
@@ -56,9 +56,9 @@ For new installs:
 
 ![image](https://user-images.githubusercontent.com/42116482/178492276-ac0f5c03-db21-482c-95e9-07dc229ac298.png)
 
-- After a few seconds, you will be automatically directed to a dashboard where you can login and view the Renovate logs. We recommend saving this [link](https://app.renovatebot.com/dashboard) for future use.
+- After a few seconds, you will be automatically redirected to the Mend's developer portal where you can sign in and view the Renovate logs. We recommend saving this [link](https://developer.mend.io) for future use.
 
-<img width="625" alt="Screen Shot 2022-07-14 at 14 24 26" src="https://user-images.githubusercontent.com/102745725/178972035-abe2e859-0756-498c-91d5-ec2f0cfdd503.png">
+<img width="600" alt="sign in page" src="https://github.com/Gabriel-Ladzaretti/tutorial/assets/97394622/0a615573-6482-44e4-a79b-07d4356ee574">
```

Comment:
@Gabriel-Ladzaretti This link points to your own fork. I was expecting the picture to be hosted on a Renovate controlled repository. Is this really what you meant?